app/src/main/java/com/jnj/vaccinetracker/common/domain/entities/Participant.kt

```diff
@@ -32,6 +32,7 @@ sealed class ParticipantBase {
     val motherLastName: String? get() = attributes[Constants.ATTRIBUTE_MOTHER_LAST_NAME]
     val childCategory: String? get() = attributes[Constants.ATTRIBUTE_CHILD_CATEGORY]
     val fullName: String get() = listOfNotNull(childFirstName, childLastName).joinToString(" ")
+    val motherName: String get() = listOfNotNull(motherFirstName, motherLastName).joinToString { "" }
```

Comment:
I've already mentioned about in previous code review. Please use correct joinToString syntax, look at the line above.